docs/src/manual/containers.md

```diff
@@ -315,13 +315,28 @@ JuMP.Containers.SparseAxisArray{Tuple{Int64, Symbol}, 1, Tuple{Int64}} with 2 en
 
 Use `eachindex` to loop over the elements:
 ```jldoctest containers_sparse
+julia> for key in eachindex(x)
+           println(x[key])
+       end
+(2, :A)
+(2, :B)
+(3, :A)
+(3, :B)
+
 julia> for key in eachindex(y)
            println(y[key])
        end
 (2, :B)
 (3, :B)
 ```
 
+!!! warning
+    In contrast to `Base.Array`s, which iterate in column-major order, that is,
+    by varying indices from left to right, `SparseAxisArray` iterates in
+    row-major order, that is, by varying the indices from right to left. As an
```

Comment:
Is this true in general or only when the `SparseAxisArray` is created by a macro? If you construct using a dict then the iteration order will match the given dict, right?

Reply:
Only when created by a macro. Dict order is as given.

Reply:
Tweaked the wording slightly.